net/nextdns/Config.in

```diff
@@ -0,0 +1,4 @@
+menu "Configuration"
+	depends on PACKAGE_nextdns
+
+endmenu
```

Comment:
Not at the moment. I may add some build flags to let the user strip the binary from some unwanted features later.

Reply:
I'm not sure what this looks like during `make menuconfig`, but if it does open an empty menu, then I would prefer to remove this for now and add it again later when there are options to select.

If `make menuconfig` is smart enough to not show an empty menu, then it would be fine to keep this as-is.

Reply:
Looks ugly, let me remove it:

<img width="764" alt="image" src="https://user-images.githubusercontent.com/68232/68803798-45011480-0615-11ea-98c0-51333ca195a2.png">